manpage/kosmorro.1.md

```diff
@@ -51,6 +51,9 @@ Available environment variables are:
 `KOSMORRO_TIMEZONE`  
     the observer's timezone (alternative to `--timezone`)
 
+`NO_COLOR`
+    disable colored console output (alternative to `--no-color`)
```

Comment:
The argument has an S in the name.
Maybe should we remove it for more convenience? I'm not a native English speaker, so I'm not sure what's the correct grammar on this :/

Reply:
I'm not sure either, but I think it's ok as is unless you would like it to be consistent with the environment variable.

Reply:
Let's keep as is for now, then :)